src/ScheduleService.Data.Provider.MsSql.Ef/Migrations/20231217210755_InitialCreate.cs

```diff
@@ -0,0 +1,141 @@
+﻿using LT.DigitalOffice.ScheduleService.Models.Db;
+using Microsoft.EntityFrameworkCore.Infrastructure;
+using Microsoft.EntityFrameworkCore.Migrations;
+using System;
+
+namespace LT.DigitalOffice.ScheduleService.Data.Provider.MsSql.Ef.Migrations;
+
+[DbContext(typeof(ScheduleServiceDbContext))]
+[Migration("20231217210755_InitialCreate")]
+public class InitialCreate : Migration
+{
+  #region Create tables
+
+  private void CreateTableGroups(MigrationBuilder migrationBuilder)
+  {
+    migrationBuilder.CreateTable(
+      name: DbGroup.TableName,
+      columns: table => new
+      {
+        Id = table.Column<Guid>(nullable: false),
+        Name = table.Column<string>(nullable: false, maxLength: 50),
+      },
+      constraints: table =>
+      {
+        table.PrimaryKey($"PK_{DbGroup.TableName}", x => x.Id);
+        //table.UniqueConstraint($"UX_{DbGroup.TableName}_Name_unique", x => x.Name);
```

Comment:
why not?

Reply:
I forgot to ask about it
Can the different group/workspace and another have the same name or something like this?
And do we need to nave this rule in db or it is need to check earlier

Reply:
+

Reply:
+